modules/mobile/app/controllers/mobile/v0/facilities_info_controller.rb

```diff
@@ -39,15 +39,20 @@ def sort(facilities, sort_method)
 
       def sort_by_recent_appointment(facilities)
         facilities.sort_by(&:name) # entries not in recent appointments will be sorted alphabetically
```

Comment:
I think this needs to be `sort_by!`. I'm also not sure this comment is correct. I believe facilities that are not in the recent appointments (in the case that there are recent appointments) will be at the bottom of the list and sorted by id.

Reply:
There should be a test that checks to make sure the leftovers are are alpha. It could be that the IDs are resulting in that situation too, but they could easily be edited to ensure it's actually alpha

The comment could be updated to make it clear they are at the bottom of the list which is intended behavior. The list should have recent appointment matches at the top sorted by most recent appointment then any facilities that don't have a match should be below those and sorted alpha.

Reply:
I left existing code untouched because my assumption was it was all operating the way it was intended. I'll get this updated